graylog2-web-interface/src/theme/utils/opacify.ts

```diff
@@ -32,10 +28,7 @@ function opacify(color: string, amount: number): string {
     return color;
   }
 
-  const parsedAlpha = chroma(color).alpha();
-  const newAlpha = (parsedAlpha * 100 + amount * 100) / 100;
-
-  return chroma(color).alpha(newAlpha).css();
+  return chroma(color).alpha(amount).css();
```

Comment:
This is now just setting the alpha for the color instead of increasing it. We should either fix the comment or adjust it to actually increast it.

Reply:
I was pretty confused on this one because it seemingly did nothing with that calculation, no matter what color I tried it on. I will adjust the comment I think it makes more sense to set the alpha, especially when you are working with hex colors.

Reply:
The calculation is overly convoluted, it could be simplified to:
```suggestion
  const newAlpha = parsedAlpha + amount;
```
So it is just adding the provided delta to the current alpha of the color. I am not sure if that is more useful. Did you check other usages of the function throughout the code? From what I can see, it is not used?